Appsilon/shiny.gosling
# Fede.16 zoom to

:warning: Review after solving #13 and #16 as this branch is based in these other ones.

Closes #17

## Changes description

*Added the zoomTo method to the `GoslingComponents` class (JS) and also the R wrapper (`zoom_to`)

## How to test
* Execute the dashboard, in the basic tab in the rule mark example select one of the chromosomes and navigate to it (see gif attached)
![Peek 2022-09-02 15-59](https://user-images.githubusercontent.com/25372718/188221244-ef9a03e0-af16-45a8-af6c-1f09fa26fa53.gif)
